src/main/java/com/faforever/client/chat/PrivateChatTabController.java

```diff
@@ -99,6 +101,8 @@ public void onChatMessage(ChatMessage chatMessage) {
       showNotificationIfNecessary(chatMessage);
       setUnread(true);
       incrementUnreadMessagesCount(1);
+      boolean chatFocused = preferencesService.getPreferences().getMainWindow().getLastView().equals(NavigationItem.CHAT.name());
+      eventBus.post(new UnreadPrivateMessageEvent(chatMessage, chatFocused));
```

Comment:
I toke a private variable that stores it. However it is no different of the preference